portfolio/src/main/java/com/google/sps/data/ShawarmaPlace.java

```diff
@@ -0,0 +1,8 @@
+package com.google.sps.data;
+
+public class ShawarmaPlace {
```

Comment:
Add some documentation to this class and its methods. Also, does it work with the fields being private?

Reply:
Yes, why not, I just convert file to array of objects and then to json string.

Reply:
Interesting. So GSON probably uses reflection to convert the JSON into the class. I asked because from a design standpoint it would be better for them to use getters/setters or accessing the fields directly, respecting their visibility. Anyway, it was just a curiosity :)

Reply:
Yes, it uses reflection so it can convert any object to json))

Reply:
I would add a comment to explain that this class is only used through reflection by Gson. Otherwise it looks weird at first to see a class containing only private fields and nothing else